src/sage/rings/polynomial/multi_polynomial_ideal.py

```diff
@@ -2812,6 +2812,21 @@ def _variety_triangular_decomposition(self, ring):
             sage: I = Ideal([ x*y - 1, (x-2)^2 + (y-1)^2 - 1])
             sage: len(I.variety())
             4
+
+        We can compute the variety of an Ideal over a Multivariate Polynomial Ring over a finite field with characteristic > 2^29, which Singular doesn't support.
+
+            sage: p = 626526183415513590854084217045148362725470879166437124330209
+            sage: F = GF(p)
+            sage: R.<x, y, z> = F[]
```

Comment:
I think the normal convention is no space after comma in `.<...>` part. (at other places there are still space around comma.)